.github/scripts/check-template-and-add-labels.ts

```diff
@@ -90,6 +92,19 @@ async function main(): Promise<void> {
   }
 
   if (labelable.type === LabelableType.Issue) {
+
+    // If labelable is a flaky test report, no template is needed (we just add a link to circle.ci in the description), we skip the template checks
+    const flakyTestsLabelFound = findLabel(labelable, flakyTestsLabel);
+    if (flakyTestsLabelFound?.id) {
+      console.log(`Issue ${labelable?.number} was created to report a flaky test. No template is needed in that case: we just add a link to circle.ci in the description. Skip template checks.`);
```

Comment:
where does the link get added?

Reply:
You made me realise this sentence wasn't very clear. I updated it in this Fixed in this [commit](https://github.com/MetaMask/metamask-extension/pull/23857/commits/723ca3353fb1c0107d60120ae4e0f69740b87269).
The idea is that flaky test issue's description is just a link redirecting to circle.ci which is why we don't expect issue's description to match [issue template](https://github.com/MetaMask/metamask-extension/blob/develop/.github/ISSUE_TEMPLATE/general-issue.yml). Example flaky test issue: https://github.com/MetaMask/metamask-extension/issues/22702